components/gc-most-requested/_screen-md-min.scss

```diff
@@ -0,0 +1,24 @@
+/*
+	Most requested component
+*/
+
+// Make sure the component is not inside a container
+main:not(.container) .gc-most-requested,
+div:not(.container) .gc-most-requested {
+	h2 {
+		float: left;
+		width: 16.666667%;
+	}
+
+	ul {
+		display: flex;
+		flex-wrap: wrap;
+		list-style-position: inside;
+		padding-left: 1.3em;
+
+		li {
+			flex-basis: 50%;
+			font-size: $small-size;
```

Comment:
Note: This is not exactly the same value used by the gc-srv-info component. May be we work later on aligning both component behaviour.